cli/commands/run/run.file.ts

```diff
@@ -18,29 +30,47 @@ export function provideRunFile(
     }
     
     console.log('parsing file data...')
-    const { transactionEvents, blockEvents, alertEvents } = getJsonFile(filePath)
+    const { transactionEvents, blockEvents, alertEvents, sequenceEvents } = getJsonFile(filePath)
 
     if (handleBlock && blockEvents?.length) {
       console.log('running block events...')
       for (const blockEvent of blockEvents) {
-        const findings = await handleBlock(blockEvent)
-        console.log(`${findings.length} findings for block ${blockEvent.hash} ${findings}`)
+        if (typeof blockEvent === 'string' || typeof blockEvent === 'number') {
+          await runHandlersOnBlock(blockEvent)
+        } else {
+          const findings = await handleBlock(blockEvent)
+          console.log(`${findings.length} findings for block ${blockEvent.hash} ${findings}`)
+        }
```

Comment:
What block events could have a type that isn't a string or a number?

Reply:
the file option allows specifying a BlockEvent json object